frontend/src/metabase-types/guards/card.ts

```diff
@@ -1,5 +1,12 @@
 import type { Card, UnsavedCard } from "metabase-types/api";
 
 export function isSavedCard(card: Card | UnsavedCard): card is Card {
-  return "id" in card && card.id != null;
+  return (
```

Comment:
We should not change this as it now may return false for actual `Card` object with negative ids